ds17r3 17keys numberpad (#25403)
qmk/qmk_firmware: keyboards/czmao/a17_ds17r3/readme.md

# mao ds 17r3 Righthand

A 17keys numberpad keyboard with RGB backlight.

> This keyboard use 16mhz HSE and APM32F103CBT6(STM32F103CBT6) as MCU.

Reviewer: > use 16mhz HSE

This is not currently configured for a 16MHz HSE, either

1. This statement is incorrect
2. The keyboard needs additional config

- Keyboard Maintainer: https://github.com/micahyy
- Hardware Supported: CZMAO
- Hardware Availability: [TaoBao](https://item.taobao.com/item.htm?ft=t&id=819874589305)

>     make czmao/ds17r3l:default
> 
> Make example for this keyboard (after setting up your build environment):
> 
>     make czmao/ds17r3r:default

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make czmao/a17_ds17r3:default

Flashing example for this keyboard:

    make czmao/a17_ds17r3:default:flash
```

Author: - Keyboard Maintainer: https://github.com/micahyy
- Hardware Supported: CZMAO
- Hardware Availability: [TaoBao](https://item.taobao.com/item.htm?ft=t&id=819874589305)

    make czmao/a17_ds17r3:default

- Flashing example for this keyboard:
    make czmao/a17_ds17r3:default:flash

Author: now it means I can't upload to the QMK ,OK  I quit

# mao ds 17r3 Righthand

> ![alt text](image.png)

Author: deleted image

A 17keys numberpad keyboard with RGB backlight.
This keyboard use 16mhz HSE and APM32F103CBT6(STM32F103CBT6) as MCU.

- Keyboard Maintainer: https://github.com/micahyy
- Hardware Supported: CZMAO
- Hardware Availability: [TaoBao](https://item.taobao.com/item.htm?ft=t&id=819874589305)

    make czmao/ds17r3l:default

Make example for this keyboard (after setting up your build environment):

    make czmao/ds17r3r:default

> 
> 

Reviewer 2: ```suggestion
```

See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).

## Bootloader

Enter the bootloader in 2 ways:

* **Bootmagic reset**: Hold down the key Number Lock and plug in the keyboard
* **Physical reset button**: Briefly short the reset keys on the back of the PCB

> 
> 
> 
> 
> 

Reviewer 2: ```suggestion
```

Author: done
